src/components/OrgPostCard/OrgPostCard.spec.tsx

```
@@ -1,821 +1,1008 @@
-          <OrgPostCard {...props} />
+          <OrgPostCard post={mockPost} />
         </I18nextProvider>
       </MockedProvider>,
     );
+  };
+  describe('Tests', () => {
+    test('clicking the "close" menu option closes the menu', async () => {
+      render(
+        <MockedProvider>
+          <I18nextProvider i18n={i18n}>
+            <MemoryRouter>
```

ESLint

[error] 'MemoryRouter' is not defined.

(react/jsx-no-undef)